assets/src/edit-story/components/library/panes/media/mediaUtils.js

```diff
@@ -14,6 +14,11 @@
  * limitations under the License.
```

Comment:
I think these media utils should live in the media provide personally.

Reply:
Looks like a general helper, right? Yeah. Makes total sense move it to mediaProvider. I will take a look at it 👍